test/tests/problems/eigen_problem/gipm.i

```diff
@@ -0,0 +1,117 @@
+[Mesh]
+  type = GeneratedMesh
+  dim = 2
+  xmin = 0
+  xmax = 100
+  ymin = 0
+  ymax = 100
+  elem_type = QUAD4
+  nx = 64
+  ny = 64
+
+  displacements = 'x_disp y_disp'
+[]
+
+#The minimum eigenvalue for this problem is 2*(pi/a)^2 + 2 with a = 100.
+#Its inverse will be 0.49950700634518.
+
+[Variables]
+  [./u]
+    # second order is way better than first order
```

Comment:
I should not copy this comment from Yaqi. I have no idea whether the SECOND is better than the FIRST order or not. I will remove this comment to avoid confusing people.